changelog.txt

```diff
@@ -1,5 +1,28 @@
 *** Changelog ***
 
+2020.06.04 - version 3.1.0
+* New: Setup wizard for new users - [#3066](https://github.com/Automattic/sensei/pull/3066), [#3085](https://github.com/Automattic/sensei/pull/3085), [#3093](https://github.com/Automattic/sensei/pull/3093), [#3082](https://github.com/Automattic/sensei/pull/3082), [#3101](https://github.com/Automattic/sensei/pull/3101), [#3103](https://github.com/Automattic/sensei/pull/3103), [#3096](https://github.com/Automattic/sensei/pull/3096), [#3117](https://github.com/Automattic/sensei/pull/3117), [#3118](https://github.com/Automattic/sensei/pull/3118), [#3119](https://github.com/Automattic/sensei/pull/3119), [#3102](https://github.com/Automattic/sensei/pull/3102), [#3157](https://github.com/Automattic/sensei/pull/3157), [#3106](https://github.com/Automattic/sensei/pull/3106), [#3121](https://github.com/Automattic/sensei/pull/3121), [#3172](https://github.com/Automattic/sensei/pull/3172), [#3175](https://github.com/Automattic/sensei/pull/3175), [#3174](https://github.com/Automattic/sensei/pull/3174), [#3176](https://github.com/Automattic/sensei/pull/3176), [#3189](https://github.com/Automattic/sensei/pull/3189), [#3184](https://github.com/Automattic/sensei/pull/3184), [#3156](https://github.com/Automattic/sensei/pull/3156), [#3181](https://github.com/Automattic/sensei/pull/3181), [#3190](https://github.com/Automattic/sensei/pull/3190), [#3239](https://github.com/Automattic/sensei/pull/3239), [#3227](https://github.com/Automattic/sensei/pull/3227), [#3198](https://github.com/Automattic/sensei/pull/3198), [#3241](https://github.com/Automattic/sensei/pull/3241), [#3194](https://github.com/Automattic/sensei/pull/3194), [#3250](https://github.com/Automattic/sensei/pull/3250), [#3246](https://github.com/Automattic/sensei/pull/3246), [#3258](https://github.com/Automattic/sensei/pull/3258)
+* New: Filter for enabling/disabling enrolment related background jobs [#3182](https://github.com/Automattic/sensei/pull/3182)
+* New: Add "Date Completed" field to the Learner Management page [#3064](https://github.com/Automattic/sensei/pull/3064) 👏 @timwiel
+* New: Add support for translating strings in JavaScript files [#3219](https://github.com/Automattic/sensei/pull/3244)
+* New: Bundle WordPress components CSS [#3226](https://github.com/Automattic/sensei/pull/3226)
```

Comment:
This one and the one below are I would consider maintenance PRs.

Reply:
https://github.com/Automattic/sensei/pull/3261/commits/7d1c07a6ff37a838877bd53dc87ed70065d2a96a